build/android/settings.gradle

```diff
@@ -1 +1 @@
-rootProject.name = "Minetest"
```

Comment:
Please reintroduce it again.
When you have a lot of projects open, and all are called `android`, I ask you how do you know which project is it ?
![image](https://user-images.githubusercontent.com/3314607/72190078-e54d9c80-33fe-11ea-95b1-54c07dba0185.png)

Reply:
True, accidentally deleted during refactoring. Thanks!